### ISSUE TITLE
Empty header cells fail A11y validation

### ISSUE THREAD

**Issue by foosbar** [blackducksoftware/backgrid#2]
Empty TH cells are frowned upon during A11y validation.  Instead, we should output empty TDs.  These occur in places where a table header doesn't need a label.
